workshops/views.py

```diff
@@ -486,6 +513,32 @@ def membership_create(request, org_domain):
     return render(request, 'workshops/generic_form.html', context)
 
 
+class MembershipCreate(OnlyForAdminsMixin, PermissionRequiredMixin,
+                 CreateViewContext):
+    permission_required = 'workshops.add_membership'
```

Comment:
The existing `membership_create` view has an extra permission required. Is that of any concern?